string id
---
bool result
float32 voxelSize
